src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -56,8 +56,9 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
             await spectralManager.InstallSpectralAsync(cancellationToken);
```

Comment:
Installing the tools can be parallelized easily while keeping the same exception handling (without AggregateException from Task.WhenAll):

> EDIT: if we do that, then our logger abstraction should be protected with a lock, as the logger isn't probably thread safe (to verify!)

```csharp
var installSwaggerCliTask = swaggerManager.InstallSwaggerCliAsync(cancellationToken);
var installSpectralTask = spectralManager.InstallSpectralAsync(cancellationToken);
var installOasdiffTask = oasdiffManager.InstallOasdiffAsync(cancellationToken);

await installSwaggerCliTask;
await installSpectralTask;
await installOasdiffTask;
```